### DISCUSSION TITLE
Track Running Application or Processes

### DISCUSSION THREAD

**Discussion by warllo54** [timmo001/system-bridge#2445] [Ideas] [answered]
Hi!

I'd like to request that system-bridge reports the running processes to Home Assistant so that I can automate against what processes are running on the pc. For example, if I'm running Adobe Photoshop turn off the main lights and turn on the lamp. Or if Zoom is running mute the Music.

Thanks for looking!

**Comment by willoma** [comment]
Hello,

I have a similar need, to track the apps my kids are using on their computers.

**Comment by timmo001** [comment, marked as answer]
https://github.com/timmo001/system-bridge/issues/2836

See comment for details of Home Assistant integration feature
